### PR TITLE
Fix: telemetry parent id when using W3C activity format

### PR DESCRIPTION
**Pull request by ylabade**
Fix Issue #2142 

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

### CODE REVIEW

**Comment by cijothomas** [comment]
@ylabade Thanks for opening the bug and the PR. We'll review this soon.

File: WEB/Src/DependencyCollector/DependencyCollector/TelemetryDiagnosticSourceListener.cs [hunk 1]
```diff
@@ -50,8 +50,23 @@ public void OnEvent(KeyValuePair<string, object> evnt, DiagnosticListener diagno
             }
 
             // properly fill dependency telemetry operation context
-            telemetry.Context.Operation.Id = currentActivity.RootId;
-            telemetry.Context.Operation.ParentId = currentActivity.ParentId;
+            if (currentActivity.IdFormat == ActivityIdFormat.W3C)
```
Comment (cijothomas) [inline WEB/Src/DependencyCollector/DependencyCollector/TelemetryDiagnosticSourceListener.cs:53]: Could you add unit test for this? There should already be unit tests for this class,

**Review by cijothomas — COMMENTED** [review]
Changes look good. Please add a unit test to cover the test gap.

File: WEB/Src/DependencyCollector/DependencyCollector/TelemetryDiagnosticSourceListener.cs (hunk 1 above)
Comment (ylabade) [reply WEB/Src/DependencyCollector/DependencyCollector/TelemetryDiagnosticSourceListener.cs:53]: I'll do it, thanks for the review

**Review by cijothomas — APPROVED** [review]

**Pull request merged**